src/bokeh/.ruff.toml

```diff
@@ -3,7 +3,7 @@ extend = "../../pyproject.toml"
 [lint.flake8-tidy-imports]
 banned-module-level-imports = [
     "ipython",
-    "pandas",
+    # "pandas", TODO exclude *.pyi files
```

Comment:
If we can't figure out how to enable this selectively, I can use `TYPE_CHECKING` in `*.pyi` files where `pandas` is mentioned.